brayns/common/scene/Scene.h

```diff
@@ -30,10 +30,37 @@
 
 #include <shared_mutex>
 
+SERIALIZATION_ACCESS(ClipPlane)
 SERIALIZATION_ACCESS(Scene)
 
 namespace brayns
 {
+
+class ClipPlane : public BaseObject
```

Comment:
move this to a separate file

Reply:
Can I still keep the definition of _nextID in Scene.cpp (creating a new file just for that is annyoing)?

Reply:
Adding a file is annoying? Having something from a different file in the scene.cpp is annoying :)

Reply:
Yes it's annoying